homeassistant/components/ambient_station/sensor.py

```diff
@@ -83,7 +83,13 @@ def unit_of_measurement(self):
             w_m2_brightness_val = self._ambient.stations[self._mac_address][
                 ATTR_LAST_DATA
             ].get(TYPE_SOLARRADIATION)
-            self._state = round(float(w_m2_brightness_val) / 0.0079)
+
+            try:
+                self._state = round(float(w_m2_brightness_val) / 0.0079)
+            except TypeError:
```

Comment:
Non parse-able floats raise ValueError

Reply:
Not in this case: https://github.com/home-assistant/home-assistant/issues/32237

Reply:
I can correct the comment to make it clearer that this happens when that data is `None`?